### DISCUSSION TITLE
Using anchors and description in a table row

### DISCUSSION THREAD

**Discussion by pguptaDS** [twilio-labs/paste#2112] [Q&A] [answered]
Is it possible to use anchors and description in a row within a table? I tried to change the variation but it seems like its not compatible together? If I detach the table and do it myself, autolayout makes it really difficult to maintain the layout of the table. 

In this example - I would like to use an anchor to write out the name and make "Dan Theurer" clickable. 
![image](https://user-images.githubusercontent.com/64559618/148168578-58dfebeb-3228-4f31-a133-5183990bd40c.png)

**Comment by pguptaDS** [comment]
I am also curious to learn how to edit the table rows efficiently? Changing the content within the table is challenging right now and I'd love some pointers.

**Comment by loreina** [comment]
Hi @pguptaDS,

For editing Table rows efficiently, our current recommendation is to detach the component as there's currently no method to add or remove layers in a component otherwise. You can detach Table to use both Anchor and description text in a body cell, and we realize this is a pain point.

We're exploring a few options to make it easier to swap components within containerized components like Table, Modal, etc. Thanks for your patience, we'll get back to you in a bit!

**Comment by loreina** [comment]
Hey @pguptaDS,

Just a quick update: DSYS is working on making some optimizations to the Table component in Figma so that the body cell content can be swappable. We'll follow up when the changes are published to Figma. Thanks!

**Comment by loreina** [comment, marked as answer]
Hi there, the Table component in Figma has variants for Anchor and description text.
